bin/internal/dart-sdk.version

```diff
@@ -1 +1 @@
-1.25.0-dev.7.0
+hash/a83a69ff3eff845980308779cadbdb80cc03a603
```

Comment:
After discussing this with @a-siva a little bit more, to unlock this cl, it seems we can leave it up to flutter developer who wants to use '--preview-dart-2' flag to update dart-sdk.version, rerun the script that downloads it themselves.
We can also pick up https://codereview.chromium.org/2982943002/ into next dart dev build, roll flutter tools to that, which would also fix '--preview-dart-2'.